src/lib/modals/moderation/ModPanel.svelte

```diff
@@ -256,11 +255,6 @@
 					on:click={() => modals.showModal(KickAllUsersModal)}
 					>Kick all users</button
 				>
```

Comment:
should be in 256 line

Reply:
if you mean the positioning of that one `>`, this is done by the formatter we're using (i assume it's trying to make some of the `</button>` be unindented without adding any white space to the element which might affect things)

Reply:
yeah thats from prettier